db/seeds.rb

```diff
@@ -22,12 +26,16 @@
 
 # Generated Users
 fake_users = (1..5).map do |index|
-  User.create!(email: "example#{ index }@example.com", username: Faker::Name.name, password: "123456", password_confirmation: "123456", confirmed_at: Time.now)
+  if foundUser = User.find_by(email: "example#{ index }@example.com")
```

Comment:
WTF? 
```ruby
if User.find_by(email: "example#{ index }@example.com").blank?
  User.create!(email: "example#{ index }@example.com", username: Faker::Name.name, password: "123456", password_confirmation: "123456", confirmed_at: Time.now)
end
```

Reply:
preteja gadijuma ja tomer ir user, tad netiek pievienots pie array ar kuru pectam strada lai veidotu posts un communities, tapec pieliku lai atgriez uz array to useri ko atrada.